src/collection-facets.ts

```diff
@@ -191,16 +193,25 @@ export class CollectionFacets extends LitElement {
       closeOnBackdropClick: true, // TODO: want to fire analytics
       title: html`Select a date range`,
     });
+
     this.modalManager?.classList.add('expanded-date-picker');
     this.modalManager?.showModal({
       config,
       customModalContent,
-      userClosedModalCallback: () => {
-        this.modalManager?.classList.remove('expanded-date-picker');
-      },
+      userClosedModalCallback: this.handleExpandedDatePickerClosed,
+    });
+
+    this.analyticsHandler?.sendEvent({
+      category: analyticsCategories.default,
+      action: analyticsActions.histogramExpanded,
+      label: 'expanded',
```

Comment:
QUESTION: Often, we put the page's URL in the `label`. Would it be more useful to put the URL or the collection identifier here?

Reply:
Done -- label is now the URL.